### ISSUE TITLE
Media type icon in controls bar?

### ISSUE THREAD

**Issue by jbenet** [athenalabs/acorn-player#2]
Would it make sense to add the media-type icon to the controls bar?

![wat-is-it?](https://img.skitch.com/20120911-egxje2b24arxmpyedmchiwjqw5.png)

**Comment by tenedor** [comment]
does this add any value except to someone looking at a screenshot of the acorn?

**Comment by jbenet** [comment]
fair point -- i think it will for later, more complicated types (galleries vs playlist vs slideshow, etc).

**Comment by tenedor** [comment]
mhm i can see that. let's keep in mind and revisit when we get there
